app/bundles/CoreBundle/Assets/css/app.css

```diff
@@ -6618,6 +6618,123 @@ a.inline-token .text-muted {
 .map-options__item.btn.active:focus {
   outline: 0 !important;
 }
+.accordion {
+  --layout-density-padding-inline-local: 16px;
+  box-sizing: border-box;
+  padding: 0;
+  border: 0;
+  margin: 0;
+  font-family: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+  inline-size: 100%;
+  list-style: none;
+}
+.accordion .panel {
+  display: list-item;
+  border: none;
+  border-radius: 0;
+  margin: 0;
+  overflow: visible;
+  -webkit-border-before: 1px solid var(--border-subtle);
```

Comment:
Generally not a good idea to target only 1 specific kind of browser for styling, like you are doing here for webkit. That's only good to use a fallback for a specific standard style that's not working with all browsers. I'm not seeing that, but maybe I missed it? 
So you should be using styling that all browsers support, alongside a possible fallback or prefix. In this case, you could use border, outline or even box-shadow to replace cases of `border-before` and `border-after` I think.

Reply:
The styling for other browsers is happening with the `border-block-start` property

`border-block-start` = `-webkit-border-before`

Reply:
Ah, I see now. In that case, do double-check if it works in Firefox, because these properties don't seem to have any support in that browser. If it doesn't, and there's not firefox prefix, you'll have to use something else for the styling.

- https://caniuse.com/?search=border-before

Reply:
I removed the webkit before property to use only border-block (-start or -end) which has support